.github/workflows/main.yml

```diff
@@ -1,29 +1,56 @@
+<<<<<<< HEAD
```

Comment:
Please remove `<<<<<<< HEAD` in your codes on line 1

Reply:
Thanks were removed